articles/active-directory/hybrid/whatis-azure-ad-connect-v2.md

```diff
@@ -102,9 +102,10 @@ This is a known issue.  To resolve this, restart your PowerShell session after i
  
  1.  Open Windows PowerShell with administrative privileges
  2.  Type or copy and paste the following: 
-    ``` powershell
-              Import-module -Name "C:\Program Files\Microsoft Azure AD Sync\Bin\ADSync"
-    ```
+ 
+ ```PowerShell
+ Import-module -Name "C:\Program Files\Microsoft Azure AD Sync\Bin\ADSync"
+ ```
```

Comment:
@YuriySamorodov - The edit option isn't available for me in this pull request. Before we merge, could you make these style edits? Thanks!

1.  Open Windows PowerShell with administrative privileges.
2.  Type or copy and paste the following code: 

     ```powershell
     Import-module -Name "C:\Program Files\Microsoft Azure AD Sync\Bin\ADSync"
     ```

Reply:
Done